tests/TestCase/Database/Schema/MysqlSchemaTest.php

```diff
@@ -254,6 +254,7 @@ public function testDescribeTable()
         $expected = [
             'id' => [
                 'type' => 'biginteger',
+                'baseType' => null,
```

Comment:
The base type defaults, when using method calls, to the same type. I will change this as it seems to cause confusion